src/repr/src/refresh_schedule.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::adt::interval::Interval;
+use crate::Timestamp;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct RefreshSchedule {
+    // `REFRESH EVERY`s
+    pub everies: Vec<RefreshEvery>,
+    // `REFRESH AT`s
+    pub ats: Vec<Timestamp>,
+}
+
+impl RefreshSchedule {
+    pub fn empty() -> RefreshSchedule {
+        RefreshSchedule {
+            everies: Vec::new(),
+            ats: Vec::new(),
+        }
+    }
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct RefreshEvery {
+    pub interval: Interval,
```

Comment:
```suggestion
    pub interval: Duration,
```

Recommend storing a `Duration` here, and converting from `Interval` to `Duration` in the SQL planner. That's what we do elsewhere. The `Interval` type is annoying as hell to use from Rust.

Reply:
Ok, done: https://github.com/MaterializeInc/materialize/pull/23870/commits/67f93859f1cd8d85ec74c3343479230a4d313225